chapter_recurrent-modern/lstm_vn.md

```diff
@@ -96,14 +101,15 @@ where $\mathbf{W}_{xi}, \mathbf{W}_{xf}, \mathbf{W}_{xo} \in \mathbb{R}^{d \time
 are weight parameters and $\mathbf{b}_i, \mathbf{b}_f, \mathbf{b}_o \in \mathbb{R}^{1 \times h}$ are bias parameters.
 -->
 
-*dịch đoạn phía trên*
+trong đó $\mathbf{W}_{xi}, \mathbf{W}_{xf}, \mathbf{W}_{xo} \in \mathbb{R}^{d \times h}$ và $\mathbf{W}_{hi}, \mathbf{W}_{hf}, \mathbf{W}_{ho} \in \mathbb{R}^{h \times h}$
+là các trọng số và $\mathbf{b}_i, \mathbf{b}_f, \mathbf{b}_o \in \mathbb{R}^{1 \times h}$ là các hệ số điều chỉnh.
 
 
 <!--
 ### Candidate Memory Cell
 -->
 
-### *dịch tiêu đề phía trên*
+### Ô Nhớ Tiềm năng
```

Comment:
```suggestion
### Ký ức Tiềm năng
```
Ở dưới có nói kí hiệu là `$\tilde{\mathbf{C}}_t $`, trong hình đã dịch là `ký ức tiềm năng`, nên em nghĩ để ký ức cho thống nhất. Để xem cả hình thì vào `File changed` rồi chọn `Display the rich diff`
![image](https://user-images.githubusercontent.com/30251584/83613204-ab4b8580-a5ad-11ea-8b89-8748417b9c3a.png)

Reply:
À okie, chắc anh quên :3